website_docs/_index.md

```diff
@@ -19,14 +16,29 @@ as follows:
 
 | Tracing | Metrics | Logging |
 | ------- | ------- | ------- |
-| Beta    | Not Yet Implemented | Not Yet Implemented |
+| Release Candidate | Not Yet Implemented | Not Yet Implemented |
 
-The current release can be found [here](https://github.com/open-telemetry/opentelemetry-ruby/releases)
+The current release can be found [here][releases]
+
+## Using OpenTelemetry Ruby
+
+- [Quick Start][quick-start]
+- [Context Propagation][context-propagation]
+- [Span Events][events]
+- [Manual Instrumentation][manual-instrumentation]
 
 ## Further Reading
 
-- [OpenTelemetry for Ruby on GitHub](https://github.com/open-telemetry/opentelemetry-ruby)
-- [Installation](https://github.com/open-telemetry/opentelemetry-ruby#installation)
-- [Quick Start](https://github.com/open-telemetry/opentelemetry-ruby#quick-start)
-- [API Documentation](https://open-telemetry.github.io/opentelemetry-ruby/)
-- [Examples](https://github.com/open-telemetry/opentelemetry-ruby/tree/main/examples)
+- [OpenTelemetry for Ruby on GitHub][repository]
+- [Ruby API Documentation][ruby-docs]
+- [Examples][examples]
```

Comment:
Maybe add a link to the broader opentelemetry project? IDK if we really need it, though.

Reply:
This is going to be hosted on `opentelemetry.io` and is intended to replace our existing documentation using a submodule so it may be rendundant?

https://opentelemetry.io/docs/ruby/
https://github.com/open-telemetry/opentelemetry.io

Reply:
Ah, makes sense. No need then, I agree. 😄